src/datepicker/datepicker-i18n.ts

```diff
@@ -28,6 +28,13 @@ export abstract class NgbDatepickerI18n {
    */
   abstract getWeekdayShortName(weekday: number): string;
 
+  /**
+   * Returns the weekday name to listen from screen reader in the heading of the month view.
+   *
+   * With default calendar we use ISO 8601: 'weekday' is 1=Monday ... 7=Sunday.
+   */
+  getWeekdayName(weekday: number): string { return this.getWeekdayShortName(weekday); }
+
```

Comment:
`NgbDatepickerI18n` is supposed to be an abstract class, with no simplementation.
Default implementation should go in `NgbDatepickerI18nDefault`

Reply:
Removed implementation from NgbDatepickerI18n. Moved method implementation to class implementations.